src/components/common/Button.tsx

```diff
@@ -10,23 +10,28 @@ interface SizeType {
   };
 }
 
-const SIZES: SizeType= {
+const SIZES: SizeType = {
   small: { borderRadius: '8', padding: '12px 23px', fontSize: '16' },
   medium: { borderRadius: '40', padding: '12px 71px', fontSize: '18' },
   large: { borderRadius: '40', padding: '16px 124px', fontSize: '20' },
 };
 
-interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement>{}
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {}
 
-export default function Button({ className, onClick, children, disabled } : ButtonProps) {
+export default function Button({
+  className,
+  onClick,
+  children,
+  disabled,
+}: ButtonProps) {
   return (
     <button className={className} disabled={disabled} onClick={onClick}>
       {children}
     </button>
   );
 }
 
-export const StyledButton = styled.button<{size: string;}>`
+export const StyledButton = styled.button<{ size: string }>`
```

Comment:
[참고]
```suggestion
export const StyledButton = styled.button<{ $size: string }>`
```
styledComponent에서 props는 실제 DOM에 값을 넣고있습니다. 이를 방지하기 위한 것이 [transientProps](https://styled-components.com/docs/api#transient-props)란 것이 있는데요, 이것은 컴포넌트에 전달된 props 중 스타일링에만 사용되고 실제 HTML 엘리먼트에 전달되지 않도록 해준답니다.

Reply:
그런데 지금 css를 해주는 파일들이 scss도 있고, css도 있고 styledComponent도 있는데 이건 하나로 통일하고 계신 과정중이실까요?!

Reply:
전에 했던것들은 css인데 프로젝트 8하면서 scss로 변경하고 싶어서 조금 섞여있습니다..!! 조금씩 수정해보겠습니당